### DISCUSSION TITLE
我按照例子用SQL建立了一个表同步任务，但当我重启flink后，数据又要重新同步一次。

### DISCUSSION THREAD

**Discussion by yyxxhh** [apache/flink-cdc#2498] [Q&A]
我按照例子用SQL建立了一个表同步任务，如insert into sink_orders select * from orders; flink也配置了savepoint和checkpoint路径。但当我重启flink，发现任务不见了，然后重新执行这个SQL，发现表的数据又会重新初始化同步一次。如果我想不重新同步，只是继续上次的任务增量同步，需要怎么做？

**Comment by jefferyjob** [comment]
同问，我也有这个问题

**Comment by chaoxxin** [comment]
同问

**Comment by banyue0618** [comment]
你的flink部署方式是Standalone-HA高可用集群模式吗，普通的单机模式是不会根据检查点恢复任务的。
